ui/components/app/modals/modal.js

```diff
@@ -117,7 +117,7 @@ const custodyConfirmModalStyle = {
 
 const MODALS = {
   ONBOARDING_ADD_NETWORK: {
-    contents: <AddNetworkModal />,
+    contents: <AddNetworkModal isNewNetworkFlow={false} />,
```

Comment:
Awesome, awesome catch!  I hadn't thought of this flow!

Could we provide a better prop name than `isNewNetworkFlow`?  At first glance I don't understand what it represents.